08_voting/index.tex

```diff
@@ -1 +1,192 @@
 \chapter{Voting}
+\section{Voting Scenarios}
+\label{sec:VotingScenarios}
+
+The voting is called when a collective decision is needed to make in a justified way. A robust voting system is designed for the organisational meetings especially for IIGO in this coursework. % TODO ref to IIGO chapter
+When a voting is called in IIGO, the roles President, Speaker, Judge, and Voters will be empowered to have relevant functions to proceed a series of action:
+\begin{itemize}
+    \item President will be eligible to select a motion for the archipelago to vote for.
+    \item Speaker will be eligible to take charge of the proceeding of voting i.e. receiving ballots, counting ballots, and declare the winner.
+    \item Judge will be eligible to inspect ballots and the Speaker's action. % ref to IIGO judge inspectBallot
+    \item Voters will be eligible to vote i.e. submit ballots.
+\end{itemize}
+
+In an IIGO meeting, the elections of roles and changes of rules need voting for such collective decision-making.
+
+\textbf{Elections of roles:} As per the IIGO specification, the elections of roles will be conducted according to the cycle of accountability i.e. the Judge can initiate the power transfer of the role of President through voting; the President can initiate the power transfer of the role of \texttt{Speaker} through voting; the Speaker can initiate the power transfer of the role of Judge through voting. To prevent multiple-role enactment of one island, the island that has been selected as a role will be removed from the candidates in the voting for other roles in the current turn. Depending on the sequence of the elections called, the candidates for three roles will consist of maximally six, five, and four islands accordingly. The ballots from the Voter islands will be in the format of preference lists, which rank all candidates in an order.
+
+\textbf{Changes of rules:} the President can select a rule from the proposed list to be voted by the Voter islands. The Speaker will host the voting for changes of rules and be responsible for the ballots counting and results declaration. The ballots from the Voter islands will be: Aye, namely agree with the rule change; Nay, namely disagree with the rule change; or Abstain.
+
+
+\section{Voting Methods}
+\label{sec:VotingMethods}
+
+Table \ref{table:votingmethod} indicates available choices for each voting scenario.
```

Comment:
Table \ref{table:votingmethod} indicates the available voting method choices for each voting scenario considered in the coursework.

Reply:
changed